SCF-Session-6/polynomial/PolyTest.java

```diff
@@ -0,0 +1,95 @@
+package polynomial;
+
+import static org.junit.Assert.*;
+
+public class PolyTest {
+
+    /**
+     * Test case when input polynomial is null
+     */
+    @org.junit.Test(expected = AssertionError.class)
+    public void polyFirstException() 
+    {
+        new Poly(null);
+    }
+    
+    
+    /**
+     * Test case when input polynomial is empty
+     */
+    @org.junit.Test(expected = AssertionError.class)
+    public void polySecondException()
+    {
+        new Poly(new int[][]{});        
+    }
+    
+    
+    /**
+     * Test case for evaluate(float value) when value is too large
+     */
+    @org.junit.Test(expected = AssertionError.class)
+    public void evaluateTestFirst()
+    {
+        Poly polynomial = new Poly(new int[][]{{2, 200}, {3, 1}, {2, 0}});
+        polynomial.evaluate(200f);
```

Comment:
Exception message should be asserted in test methods if exception is expected.

Reply:
done